2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -830,7 +830,7 @@
  {
   "input": "Also on a recent Patreon post, the channel supporter Daksha Vaid-Quinter brought my attention to a completely different approach I hadn't seen before, which leverages the use of entropy, and again for the theoretically curious among you I'll leave some links in the description.",
   "model": "nmt",
-  "translatedText": "También en una publicación reciente de Patreon, Daksha Vaid-Quinter, partidaria del canal, me llamó la atención sobre un enfoque completamente diferente que no había visto antes, que aprovecha el uso de la entropía, y nuevamente, para los teóricamente curiosos, dejaré algunos enlaces. en la descripción.",
+  "translatedText": "También en una publicación reciente de Patreon, Daksha Vaid-Quinter, mecenas del canal, me llamó la atención sobre un enfoque completamente diferente que no había visto antes, que aprovecha el uso de la entropía, y nuevamente, para los curiosos de la teoría, voy a dejar algunos enlaces en la descripción.",
```

Again: what is wrong with "dejaré"? Unlike with the previous two words, this time I have a stronger preference towards "dejaré" (over "voy a dejar").

```suggestion
  "translatedText": "También en una publicación reciente de Patreon, Daksha Vaid-Quinter, mecenas del canal, me llamó la atención sobre un enfoque completamente diferente que no había visto antes, que aprovecha el uso de la entropía, y nuevamente, para los curiosos de la teoría, dejaré algunos enlaces en la descripción.",
```

---

I think this is a Castilian vs American Spanish thing. I'll update it, after all most Spanish speakers on YouTube are American.